view/src/components/surplus.js

```diff
@@ -350,13 +408,23 @@ class Surplus extends Component {
     if (this.state.uiLoading === true) {
       return (
         <main className={classes.content}>
-          {this.state.uiLoading && <CardSkeletons classes={classes} />}
+          {this.state.uiLoading && (
+            <CardSkeletons classes={classes} noPadding={!this.props.main} />
+          )}
         </main>
       );
     } else {
       return (
         <main className={classes.content}>
-          <div className={classes.toolbar} />
+          {/* Only show if it is not the main page */}
+          {!this.props.main && (
+            <Typography className={(classes.instructions, classes.formText)}>
+              Please select a Surplus Object that you would like to pair with a
+              Food Bank. If you would like to create a new Surplus Object, press
+              the icon in the bottom right.
```

Comment:
`the icon` and `bottom right` make a lot of assumptions about your user.

Suggest naming what the icon is/does ("add" or "edit" or whatever). And don't give directions that assume the user can tell where the bottom right is - that's entirely unhelpful to someone who can't see your screen. Also for apps that are localized, this would be wrong in RTL languages.